<ui version="4.0" >
 <class>GaussianBeamForm</class>
 <widget class="QWidget" name="GaussianBeamForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Gaussian Beam</string>
  </property>
  <layout class="QHBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Wavelength</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_Wavelength" >
         <property name="suffix" >
          <string> nm</string>
         </property>
         <property name="decimals" >
          <number>0</number>
         </property>
         <property name="minimum" >
          <double>1.000000000000000</double>
         </property>
         <property name="maximum" >
          <double>6000.000000000000000</double>
         </property>
         <property name="singleStep" >
          <double>10.000000000000000</double>
         </property>
         <property name="value" >
          <double>532.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QPushButton" name="pushButton_Add" >
         <property name="text" >
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Remove" >
         <property name="text" >
          <string>Remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QToolBox" name="toolBox" >
       <property name="minimumSize" >
        <size>
         <width>200</width>
         <height>16</height>
        </size>
       </property>
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Plain</enum>
       </property>
       <property name="currentIndex" >
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_IO" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>704</width>
          <height>426</height>
         </rect>
        </property>
        <attribute name="label" >
         <string>Display</string>
        </attribute>
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QGroupBox" name="groupBox" >
           <property name="title" >
            <string>Optics bench</string>
           </property>
           <layout class="QGridLayout" >
            <item row="0" column="0" >
             <widget class="QLabel" name="label_5" >
              <property name="text" >
               <string>Horizontal range</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QDoubleSpinBox" name="doubleSpinBox_HRange" >
              <property name="suffix" >
               <string> mm</string>
              </property>
              <property name="minimum" >
               <double>1.000000000000000</double>
              </property>
              <property name="maximum" >
               <double>100000.000000000000000</double>
              </property>
              <property name="singleStep" >
               <double>100.000000000000000</double>
              </property>
              <property name="value" >
               <double>700.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QLabel" name="label_8" >
              <property name="text" >
               <string>Vertical range</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QDoubleSpinBox" name="doubleSpinBox_VRange" >
              <property name="suffix" >
               <string> µm</string>
              </property>
              <property name="minimum" >
               <double>1.000000000000000</double>
              </property>
              <property name="maximum" >
               <double>100000.000000000000000</double>
              </property>
              <property name="singleStep" >
               <double>50.000000000000000</double>
              </property>
              <property name="value" >
               <double>5000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="0" >
             <widget class="QLabel" name="label_4" >
              <property name="text" >
               <string>Horizontal offset</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1" >
             <widget class="QDoubleSpinBox" name="doubleSpinBox_HOffset" >
              <property name="suffix" >
               <string> mm</string>
              </property>
              <property name="minimum" >
               <double>-100000.000000000000000</double>
              </property>
              <property name="maximum" >
               <double>100000.000000000000000</double>
              </property>
              <property name="singleStep" >
               <double>100.000000000000000</double>
              </property>
              <property name="value" >
               <double>-100.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2" >
           <property name="title" >
            <string>File</string>
           </property>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QPushButton" name="pushButton_Open" >
              <property name="text" >
               <string>Open</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QPushButton" name="pushButton_Save" >
                <property name="text" >
                 <string>Save</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_SaveAs" >
                <property name="text" >
                 <string>Save as...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_Print" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="text" >
               <string>Print</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Refresh" >
           <property name="text" >
            <string>Refresh</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_MagicWaist" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>144</width>
          <height>330</height>
         </rect>
        </property>
        <attribute name="label" >
         <string>Magic waist</string>
        </attribute>
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>Target waist</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_TargetWaist" >
           <property name="suffix" >
            <string> µm</string>
           </property>
           <property name="maximum" >
            <double>100000.000000000000000</double>
           </property>
           <property name="value" >
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6" >
           <property name="text" >
            <string>Tolerance</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_WaistTolerance" >
           <property name="suffix" >
            <string> %</string>
           </property>
           <property name="decimals" >
            <number>1</number>
           </property>
           <property name="minimum" >
            <double>0.000000000000000</double>
           </property>
           <property name="maximum" >
            <double>100000.000000000000000</double>
           </property>
           <property name="value" >
            <double>5.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Target position</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_TargetPosition" >
           <property name="suffix" >
            <string> mm</string>
           </property>
           <property name="minimum" >
            <double>-100000.000000000000000</double>
           </property>
           <property name="maximum" >
            <double>100000.000000000000000</double>
           </property>
           <property name="value" >
            <double>550.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7" >
           <property name="text" >
            <string>Tolerance</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_PositionTolerance" >
           <property name="suffix" >
            <string> % z0</string>
           </property>
           <property name="decimals" >
            <number>1</number>
           </property>
           <property name="maximum" >
            <double>100000.000000000000000</double>
           </property>
           <property name="value" >
            <double>5.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_Scramble" >
           <property name="text" >
            <string>Scramble lenses</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_MagicWaist" >
           <property name="text" >
            <string>Find</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_WaistResult" >
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_Fit" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>150</width>
          <height>283</height>
         </rect>
        </property>
        <attribute name="label" >
         <string>Waist fit</string>
        </attribute>
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QComboBox" name="comboBox_FitData" >
           <property name="currentIndex" >
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="fitTable" >
           <property name="maximumSize" >
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Fit" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="text" >
            <string>Fit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_FitResult" >
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_SetInputBeam" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>Set as input beam</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_SetTargetBeam" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>Set as target beam</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QSplitter" name="splitter" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTableView" name="table" />
       <widget class="QWidget" native="1" name="widget" />
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Status" >
       <property name="toolTip" >
        <string/>
       </property>
       <property name="frameShape" >
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="action_AddLens" >
   <property name="text" >
    <string>&amp;Lens</string>
   </property>
  </action>
  <action name="action_AddFlatInterface" >
   <property name="text" >
    <string>&amp;Flat Interface</string>
   </property>
  </action>
  <action name="action_AddCurvedInterface" >
   <property name="text" >
    <string>&amp;Cuved Interface</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="GaussianBeam.qrc" />
 </resources>
 <connections/>
</ui>
